docs/team/glemenneo.md

```diff
@@ -11,6 +11,11 @@ WorkBook (WB) is a desktop app for CS/tech students who are applying for interns
 
 ### Summary of contributions
 
+Updated UG from AB3 to WB.
+Redesigned Internship card to better fit WB.
+Implemented the Tips window and Tips button.
+Implemented Responsive UI. 
+
```

Comment:
I'll approve first but rmb to elaborate and state all the good work you've done by next week!!